src/subscription_manager/gui/registergui.py

```diff
@@ -796,6 +854,31 @@ def choose_initial_screen(self):
         return False
 
 
+class FirstbootWidget(RegisterWidget):
+    # A RegisterWidget for use in firstboot.
+    # This widget, along with the PerformForceRegisterScreen, will ensure that
+    # all screens are shown, and the user is allowed to register again.
+    __gtype_name__ = "FirstbootWidget"
+
+    initial_screen = CHOOSE_SERVER_PAGE
```

Comment:
If RegisterWidget.choose_initial_screen starts sending UI to ConfirmUnregisterScreen, we potentially don't need this class at all.

Though I'm not sure if ChooseServerScreen should be before or after ConfirmUnregister/UnregisterIfNeededScreen. I could see either scenario.